apps/nowcasting-app/components/helpers/data.ts

```diff
@@ -74,3 +74,57 @@ export const generateGeoJsonForecastData: (
 
   return { forecastGeoJson };
 };
+
+export const filterHistoricDataCompact = <
+  T extends { datetimeUtc: string; generationKwByGspId?: any; forecastValues?: any }
+>(
```

Comment:
Is it worth adding some comments about what the functions do - https://tsdoc.org/ - even if its just one sentence

Reply:
Like, its not immediately obvious what `getHistoricBackwardIntervalMinutes` does

Reply:
I agree with @peterdudfield that adding some comments for the functions could be useful. 

I'm also finding the word `Compact` at the end of the functions a little bit confusing and wonder if the naming could be `filterCompactHistoricData` or `filterCompactFutureData` in the same way  `compact` is used for `getOldestTimestampFromCompactForecastValues`.

Reply:
Fair point, think I prefer the `filterCompactFutureData` version, have amended both function names 👍